app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,18 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            skill_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self.skill_level = skill_level
+
+    def get_rating(self) -> int:
+        return self.skill_level
```

Comment:
Ensure that the `get_rating` method accesses the `_skill_level` attribute if you change its name to be protected.